scripts/modules/b2b-account/account-hierarchy.js

```diff
@@ -0,0 +1,140 @@
+        initialize: function () {
+            Backbone.MozuView.prototype.initialize.apply(this, arguments);
+        },
+        fixMargins: function () {
+            $(document).ready(function () {
+                //close menu if we click on outside of menu
+                $(document).on("click", function (event) {
+                    if (!$(event.target).closest(".dropdown-content").length && !$(event.target).closest(".three-dots").length) {
+                        $(".account-hierarchy .dropdown-content").removeClass("active");
+                    }
+                });
+
+                //fix margins
+                $(".node:not(:has(.caret))").css("margin-left", "-10px");
+
+                //open first node of the tree always
+                $(".account-hierarchy>ul.tree li").first().children(".node").children(".caret").addClass("caret-down");
+                $(".account-hierarchy>ul.tree li").first().children(".nested").addClass("active");
+            });
+        },
+        toggleNodes: function (e) {
+            $(e.currentTarget).parent().parent().children(".nested").toggleClass("active");
+            $(e.currentTarget).toggleClass("caret-down");
+        },
+        showThreeDotsMenu: function (e) {
+            $(e.currentTarget).closest('li').children('.dropdown-content').toggleClass("active");
+        },
+        addChildAccount: function (e) {
+            //todo: need to implement this method in future.
+        },
+        expandAll: function (e) {
+            $(".tree .caret").addClass("caret-down");
+            $(".nested").addClass("active");
+        },
+        collapseAll: function (e) {
+            $(".tree .caret").removeClass("caret-down");
+            $(".nested").removeClass("active");
+        },
+        viewAccount: function (e) {
+            //todo: need to implement this method in future.
+            var accountId = e.currentTarget.dataset.mzValue;
+        },
+        changeParentAccount: function (e) {
+            //todo: need to implement this method in future.
+            var accountId = e.currentTarget.dataset.mzValue;
+        }
+    });
+
+    var AccountHierarchyModel = Backbone.MozuModel.extend({
+        mozuType: 'accounthierarchy',
+        initialize: function () {
+            var self = this;
+            self.initApiModel();
+            self.apiGet({ id: require.mozuData('user').accountId }).then(function (response) {
+                self.set("hierarchy", self.processAccountHierarchy(response.data));
+                self.syncApiModel();
+            });
+            self.set("isUserAdmin", self.isUserAdmin());
+        },
+        isUserHavingBehavior: function (behaviorId) {
+            var behaviors = require.mozuData('user').behaviors;
+            if (behaviors) {
+                return behaviors.includes(behaviorId);
+            }
+            return false;
+        },
+        isUserAdmin: function () {
+            //Only admin should have 1000 behavior.
```

Comment:
Can you add the behavior name to the comment? What is 1000? Same with the others.